Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
Done the check with 96a5d8fc6

About launchpad: Sorry, I'm not that familiar with Debian/Ubuntu and don't really understand what you say about Launchpad and PPAs.